### PR TITLE
updated to reflect the new admin ui

### LINKED ISSUE #1484: Using external storage chapter
### Description

Updated this Server Administration Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1484

### CODE REVIEW

File: server_admin/topics/users/proc-allow-user-to-delete-account.adoc [hunk 1]
```diff
@@ -23,12 +23,20 @@ You can give specific users a role that allows account deletion.
 . Click *Users* in the menu.
 . Select a user.
 . Click the *Role Mappings* tab.
+ifeval::[{project_community}==true]
+. Click the *Assign role* button.
+. Filter by origin select *account*
+. Select *delete-account*.
```
Comment (andymunro) [inline server_admin/topics/users/proc-allow-user-to-delete-account.adoc]:
```suggestion
. Click *account delete-account*.
```

File: server_admin/topics/users/proc-configuring-user-attributes.adoc [hunk 2]
```diff
@@ -19,7 +19,9 @@ image:{project_images}/user-attributes.png[]
 . Click the *Attributes* tab.
 . Enter the attribute name in the *Key* field.
 . Enter the attribute value in the *Value* field.
+ifeval::[{project_product}==true]
 . Click *Add*.
+endif::[]
```
Comment (andymunro) [inline server_admin/topics/users/proc-configuring-user-attributes.adoc]:
```suggestion
ifeval::[{project_product}==true]
. Click *Add an attribute*.
endif::[]
. Enter the attribute name in the *Key* field.
. Enter the attribute value in the *Value* field.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @edewit. Just a few changes to suggest.

**Comment by andymunro** [comment]
Thanks, @edewit. Another one  ready to merge, @ssilvert

**Comment by andymunro** [comment]
Thanks, @edewit.   Another one ready to merge, @ssilvert.

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**